attributes/default.rb

```diff
@@ -18,5 +18,5 @@
 # limitations under the License.
 #
 
-default['gluster']['version'] = '3.4'
+default['gluster']['version'] = '3.6'
```

Comment:
Looks like these are both provided by the `userspace-rcu` package in EPEL

http://www.rpmfind.net/linux/RPM/epel/7/x86_64/u/userspace-rcu-0.7.9-1.el7.x86_64.html